aws/README.md

```diff
@@ -62,64 +71,208 @@ docker build -t aws \
 -f Dockerfile .
 ```
 
-If you want to build a CentOS based custom AMI then you'll need to first
-accept the terms by visiting this [link](https://aws.amazon.com/marketplace/pp?sku=bz4vuply68xrif53movwbkpnl)
+- Note down your newly created AMI_ID and export it via `PODVM_AMI_ID` env variable
 
-Once done, run the following command:
+Once the image creation is complete, you can use the following CLI command as well to
+get the AMI_ID. The command assumes that you are using the default AMI name: `peer-pod-ami`
 
 ```
-docker build -t aws \
---secret id=AWS_ACCESS_KEY_ID \
---secret id=AWS_SECRET_ACCESS_KEY \
---build-arg AWS_REGION=${AWS_REGION} \
---build-arg BINARIES_IMG=quay.io/confidential-containers/podvm-binaries-centos-amd64 \
---build-arg PODVM_DISTRO=centos \
--f Dockerfile .
+export PODVM_AMI_ID=$(aws ec2 describe-images --query "Images[*].[ImageId]" --filters "Name=name,Values=peer-pod-ami" --region ${AWS_REGION} --output text)
+echo ${PODVM_AMI_ID}
 ```
 
->Note: images for CentOS Stream haven't been built since the release 0.8.0 of this project.
+## Build CAA container image
 
-- Note down your newly created AMI_ID
+> **Note**: If you have made changes to the CAA code and you want to deploy those changes then follow [these instructions](https://github.com/confidential-containers/cloud-api-adaptor/blob/main/install/README.md#building-custom-cloud-api-adaptor-image) to build the container image from the root of this repository.
 
-Once the image creation is complete, you can use the following CLI command as well to
-get the AMI_ID. The command assumes that you are using the default AMI name: `peer-pod-ami`
+If you would like to deploy the latest code from the default branch (`main`) of this repository then expose the following environment variable:
+
+```bash
+export registry="quay.io/confidential-containers"
+```
+
+## Deploy Kubernetes using EKS
+
+- Create cluster config file
+This config will create a single node cluster. Feel free to change it as per your requirement.
 
 ```
-aws ec2 describe-images --query "Images[*].[ImageId]" --filters "Name=name,Values=peer-pod-ami" --region ${AWS_REGION} --output text
+cat > ekscluster-config.yaml <<EOF
```

Comment:
maybe it worth considering having some templates folder and then using something like `envsubst < tmps/eks-config.yaml| oc apply -f-`

Reply:
Yeah, either this or some sed fu to make changes